code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -521,23 +521,16 @@
 	var/datum/effect/effect/system/spark_spread/s = new /datum/effect/effect/system/spark_spread
 	s.set_up(2, 1, location)
 	s.start()
-	for(var/mob/living/carbon/M in viewers(world.view, location))
-		switch(get_dist(M, location))
-			if(0 to 3)
-				if(hasvar(M, "glasses"))
-					if(istype(M:glasses, /obj/item/clothing/glasses/sunglasses))
-						continue
-
-				M.flash_eyes()
-				M.Weaken(15)
-
-			if(4 to 5)
-				if(hasvar(M, "glasses"))
-					if(istype(M:glasses, /obj/item/clothing/glasses/sunglasses))
-						continue
-
-				M.flash_eyes()
-				M.Stun(5)
+	for(var/mob/living/carbon/human/M in viewers(world.view, location))
```

Comment:
IDK if I should be particularly standards-picky about us putting in changes like this that are relatively unrelated to the PR as a whole, unless I'm misunderstanding the need for this here.

Reply:
-this is the sunglasses, isn't it.

Reply:
It's replacing the typecheck with casting then checking flash protection. Sunglasses actually don't exist currently.

Reply:
Yep, my guess was you were replacing the sunglasses check because they're gone.